<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="1" column="1">
     <widget class="QGroupBox" name="weibullDistributionGroupBox">
      <property name="title">
       <string>Распределение Вейбулла</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="muGroupBox">
         <property name="title">
          <string>μ — интенсивность</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="muMinTextLabel">
            <property name="text">
             <string>μ&lt;sub&gt;min&lt;/sub&gt;</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="muMinLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="muMaxTextLabel">
            <property name="text">
             <string>μ&lt;sub&gt;max&lt;/sub&gt;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="muMaxLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="muTextLabel">
            <property name="text">
             <string>μ</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="muDoubleSpinBox">
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="uniformDistributionGroupBox">
      <property name="title">
       <string>Равномерное распределение</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="lambdaGroupBox">
         <property name="title">
          <string>λ — интенсивность</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="lambdaMinTextLabel">
            <property name="text">
             <string>λ&lt;sub&gt;min&lt;/sub&gt;</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lambdaMinLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lambdaMaxTextLabel">
            <property name="text">
             <string>λ&lt;sub&gt;max&lt;/sub&gt;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lambdaMaxLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lambdaTextLabel">
            <property name="text">
             <string>λ</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="lambdaDoubleSpinBox">
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="sigmaLambdaGroupBox">
         <property name="title">
          <string>σ(λ) — среднеквадратичное отклонение</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="sigmaLambdaMinTextLabel">
            <property name="text">
             <string>σ&lt;sub&gt;min&lt;/sub&gt;(λ)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="sigmaLambdaMinLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="sigmaLambdaMaxTextLabel">
            <property name="text">
             <string>σ&lt;sub&gt;max&lt;/sub&gt;(λ)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="sigmaLambdaMaxLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="sigmaLambdaTextLabel">
            <property name="text">
             <string>σ(λ)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="sigmaLambdaDoubleSpinBox">
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QGroupBox" name="outputParametersGroupBox">
      <property name="title">
       <string>Выходные параметры</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="2" column="0">
        <widget class="QLabel" name="actualAverageWaitingTimeTextLabel">
         <property name="text">
          <string>Фактическое t̅&lt;sub&gt;ож&lt;/sub&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="actualAverageWaitingTimeLineEdit"/>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QPushButton" name="calculatePushButton">
         <property name="text">
          <string>Моделировать</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="estimatedAverageWaitingTimeLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="estimatedAverageWaitingTimeTextLabel">
         <property name="text">
          <string>Расчётное t̅&lt;sub&gt;ож&lt;/sub&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QGroupBox" name="ffeGroupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>ПФЭ</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="2" column="2">
        <widget class="QGroupBox" name="adequacyVarGroupBox">
         <property name="title">
          <string>Дисперсия адекватности</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QLineEdit" name="adequacyVarLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" colspan="4">
        <widget class="QTableWidget" name="ffeTableWidget">
         <column>
          <property name="text">
           <string>№</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>x1</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>x2</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>x3</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ȳ</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>σ²</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Частично нелинейная</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Разница ЧН</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Линейная</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Разница Л</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="cochranTestGroupBox">
         <property name="title">
          <string>Критерий Кохрена</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QLineEdit" name="cochranTestLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QGroupBox" name="reproducibilityVarGroupBox">
         <property name="title">
          <string>Дисперсия воспроизводимости</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QLineEdit" name="reproducibilityVarLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QGroupBox" name="fTestGroupBox">
         <property name="title">
          <string>Критерий Фишера</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <widget class="QLineEdit" name="fTestLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QGroupBox" name="coefficientsGroupBox">
         <property name="title">
          <string>Коэффициенты</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_11">
          <item row="0" column="6">
           <widget class="QGroupBox" name="a23GroupBox">
            <property name="title">
             <string>a23</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_18">
             <item row="0" column="0">
              <widget class="QLineEdit" name="a23LineEdit"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QGroupBox" name="a3GroupBox">
            <property name="title">
             <string>a3</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_15">
             <item row="0" column="0">
              <widget class="QLineEdit" name="a3LineEdit"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QGroupBox" name="a12GroupBox">
            <property name="title">
             <string>a12</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_16">
             <item row="0" column="0">
              <widget class="QLineEdit" name="a12LineEdit"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QGroupBox" name="a2GroupBox">
            <property name="title">
             <string>a2</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_14">
             <item row="0" column="0">
              <widget class="QLineEdit" name="a2LineEdit"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QGroupBox" name="a1GroupBox">
            <property name="title">
             <string>a1</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_13">
             <item row="0" column="0">
              <widget class="QLineEdit" name="a1LineEdit"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QGroupBox" name="a0GroupBox">
            <property name="title">
             <string>a0</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_12">
             <item row="0" column="0">
              <widget class="QLineEdit" name="a0LineEdit"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QGroupBox" name="a13GroupBox">
            <property name="title">
             <string>a13</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_17">
             <item row="0" column="0">
              <widget class="QLineEdit" name="a13LineEdit"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="7">
           <widget class="QGroupBox" name="a123GroupBox">
            <property name="title">
             <string>a123</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_19">
             <item row="0" column="0">
              <widget class="QLineEdit" name="a123LineEdit"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
